Signal/src/call/PeerConnectionClient.swift

```diff
@@ -70,12 +70,24 @@ class PeerConnectionClient: NSObject, RTCPeerConnectionDelegate, RTCDataChannelD
              dataChannelSignaling = "signaling"
     }
 
+    // A state in this class should only be accessed on this queue in order to
+    // serialize access.
+    //
+    // This queue is also used to perform expensive calls to the WebRTC API.
+    private static let signalingQueue = DispatchQueue(label: "CallServiceSignalingQueue")
```

Comment:
* `signalingQueue` is now a private implementation detail of this class.  
* `PeerConnectionClient`'s properties are now private and `PeerConnectionClient` takes sole responsibility for ensuring that it's state is only accessed on `signalingQueue`.